transport/http/src/main/java/org/kaazing/gateway/transport/http/bridge/filter/HttpResponseDecodingState.java

```diff
@@ -257,7 +257,8 @@ protected DecodingState finishDecode(List<Object> childProducts,
 						// create the cookie name and value
 						DefaultHttpCookie cookie = new DefaultHttpCookie(
 								nvPair[0].trim());
-						cookie.setValue(nvPair[1].trim());
+						if (nvPair.length > 1)
+						    cookie.setValue(nvPair[1].trim());
```

Comment:
@cmebarrow I have added the unit test that covers this fix in HttpResponseDecoderTest. I think we no longer need the k3po test, as the unit test seems sufficient, and the k3po would just be redundant.